application/models/Project_model.php

```diff
@@ -70,50 +64,57 @@ function get_projects($params)
 
     function get_lastest_releases($limit=10)
     {
-        $sql = 'SELECT p.id, p.title_prefix, p.title, p.url_librivox, p.description, p.project_type, p.coverart_thumbnail, p.date_catalog, COALESCE(l.two_letter_code, l.three_letter_code) AS language_code, language
-                FROM projects p 
-                JOIN languages l ON (l.id = p.language_id) 
-                WHERE p.status = "complete" 
-                ORDER BY p.date_catalog DESC 
-                LIMIT 0, '. $limit ;
+        $sql = sprintf(
+            'SELECT p.id, p.title_prefix, p.title, p.url_librivox, p.description, p.project_type, p.coverart_thumbnail, p.date_catalog, COALESCE(l.two_letter_code, l.three_letter_code) AS language_code, language
+            FROM projects p
+            JOIN languages l ON (l.id = p.language_id)
+            WHERE p.status = "complete"
+            ORDER BY p.date_catalog DESC
+            LIMIT 0, %d', $limit);
 
         $query = $this->db->query($sql);
-        return $query->result();                
+        return $query->result();
     }
 
     function get_projects_summary($user_id, $status='active')
```

Comment:
I think this function isn't used anywhere, the only references I can find are in `controllers/private/Stats`, and they're commented out. Perhaps the better fix here is to just remove the function?

Reply:
Yep, done.